spanner/tests/unit/test_keyset.py

```diff
@@ -119,6 +117,33 @@ def test_to_pb_w_start_open_and_end_closed(self):
         self.assertEqual(len(krange_pb.end_closed), 1)
         self.assertEqual(krange_pb.end_closed.values[0].string_value, KEY_2[0])
 
+    def test_to_pb_single_key_start_closed(self):
+        from google.cloud.spanner_v1.proto.keys_pb2 import KeyRange
+
+        KEY_1 = [u'key_1']
+        with self.assertRaises(ValueError):
+            krange = self._make_one(start_closed=KEY_1)
```

Comment:
1. You don't need to assign to `krange` here (Pylint would even complain that it is assigned but never used)
1. Please stop using `krange` as a variable name

Reply:
removed.